src/convertToObject.js

```diff
@@ -6,7 +6,17 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const stringArray = sourceString.split(';');
+  const filterArray = stringArray.filter((item) => item.trim().length);
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have same indentation. This makes code much more readable. So, you should write `filter` method in a new line with the same indentation as `split`.